lib/bambora/profile.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Bambora
+  class Profile < JSONRequest
+    def initialize(client)
+      super
+
+      @path = '/v1/profiles'
```

Comment:
would probably start to break into api versions here and namespacing the classes into `Bambora::V1::` in case users would want to stay on old version if supported and someone would develop V2 if available.